tests/models/imagebind/test_modeling_imagebind.py

```diff
@@ -0,0 +1,1008 @@
+        )
+
+        expected_logits_per_image_with_text = torch.tensor(
+            [[23.6142, 19.1165, 13.2448], [12.1343, 23.4165, 11.8823], [15.8471, 20.1186, 24.8246]],
+            device=torch_device,
+        )
+
+        self.assertTrue(torch.allclose(outputs_vision_audio.logits_per_audio, expected_logits_per_audio, atol=1e-4))
+        self.assertTrue(
+            torch.allclose(outputs_vision_text.logits_per_image, expected_logits_per_image_with_text, atol=1e-4)
+        )
+
+        del model
+
+        torch.manual_seed(0)
+        config = ImageBindConfig()
+        model = ImageBindModel(config).to(torch_device)
+        model.eval()
+
+        tokenizer = CLIPTokenizer.from_pretrained("openai/clip-vit-large-patch14")
+        image_processor = ImageBindImageProcessor()
+        feature_extractor = ImageBindFeatureExtractor()
+        processor = ImageBindProcessor(image_processor, tokenizer, feature_extractor)
+
+        inputs_audio_vision = processor(images=images, audio=audios, return_tensors="pt").to(torch_device)
+        inputs_text_vision = processor(images=images, text=texts, return_tensors="pt", padding=True).to(torch_device)
+
+        expected_input_features = torch.tensor(
+            [
+                [-1.2776, -0.9167, -1.2776],
+                [-1.2439, -0.8372, -0.8748],
+                [-1.1235, -0.7492, -1.0867],
+            ],
+            device=torch_device,
+        )
+
+        expected_pixel_values = torch.stack([original_image_processor(image) for image in images]).to(torch_device)
+
+        assert torch.allclose(inputs_audio_vision["pixel_values"], expected_pixel_values, atol=1e-4)
+        assert torch.allclose(inputs_audio_vision["input_features"][:, :, 0, 0, 0], expected_input_features, atol=1e-4)
+
+        expected_output_vision = torch.tensor(
+            [
+                [0.0217, -0.0969, -0.0044, -0.0203, 0.0178],
+                [0.0347, -0.0987, -0.0190, -0.0034, 0.0352],
+                [0.0389, -0.0910, -0.0230, -0.0072, 0.0455],
+            ],
+            device=torch_device,
+        )
+        expected_output_text = torch.tensor(
+            [
+                [-0.1995, 0.2042, 0.7407, 0.5275, -0.4482],
+                [-0.1800, 0.2736, 0.5057, 0.4819, -0.5618],
+                [-0.2461, 0.2926, 0.4936, 0.4322, -0.2178],
+            ],
+            device=torch_device,
+        )
+        expected_output_audio = torch.tensor(
+            [
+                [-0.0882, -0.4557, 0.3396, 1.1183, -0.0692],
+                [-0.4186, -0.2179, 0.0913, 0.9061, -0.0390],
+                [-0.1190, -0.5368, 0.2956, 1.1277, 0.0037],
+            ],
+            device=torch_device,
+        )
+
+        outputs_text_vision = model(**inputs_text_vision)
+        outputs_audio_vision = model(**inputs_audio_vision)
+
+        assert torch.allclose(outputs_text_vision.image_embeds[:, :5], expected_output_vision, atol=1e-4)
```

Comment:
I recommend 
```suggestion
        assert torch.allclose(outputs_text_vision.image_embeds[:, :5], expected_output_vision, atol=1e-3)
```
as I've noticed flakyness on some local tests